.github/workflows/python-package-daily.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11.0rc1']
+        python-version: ['3.10', '3.11.0rc1']
```

Comment:
Hmm, I wonder why this one is `3.11.0rc1`

Reply:
Oh I thought you'd know. I will remove that to see...